aries-site/src/pages/learn/grid-fundamentals-part-one.mdx

```diff
@@ -0,0 +1,198 @@
+- Allow the number of grid columns to adapt to a variety of screen widths.
+- Allow the items to adapt and flex within the available space.
+
+At the end of this tutorial, you will have built this:
+
+<!-- show example of product cards in a grid -->
+
+## Prerequisites
+
+This tutorial can be completed in its entirety by following the steps below. However,
+because Grommet is a ReactJS-based UI library, introductory knowledge of
+[React](https://reactjs.org/docs/getting-started.html) may be helpful in understanding
+the syntax.
+
+## Getting started
+
+Let’s get started. Use this [CodeSandbox template]() as a starter.
+
+First, in `App.js` add the Grid to your project by importing it from Grommet and
+placing it around the “hello world” text.
+
+```js
+// App.js
+import { Grommet, Box, Heading, Text, Grid } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+...
+return (
+  <Grommet theme={hpe}>
+    <Grid>
+      <Text>hello world</Text>
+    </Grid>
+  </Grommet>
+);
+```
+
+<br />
+
+Let's also add a heading to the page.
+
+```js
+<Grommet theme={hpe}>
+  <Heading level="1">Products</Heading>
+  <Grid>
+    <Text>hello world</Text>
+  </Grid>
+</Grommet>
+```
+
+## Add placeholder content
+
+Now that you have the Grid in place, let's add some placeholder content. Replace the
+"hello world" text with the following:
+
+```js
+<Grid>
+  <Box background="blue" border pad="medium" align="center">
+    <Text size="large">Product 1</Text>
+  </Box>
+  <Box background="blue" border pad="medium" align="center">
+    <Text size="large">Product 2</Text>
+  </Box>
+  <Box background="blue" border pad="medium" align="center">
+    <Text size="large">Product 3</Text>
+  </Box>
+  <Box background="blue" border pad="medium" align="center">
+    <Text size="large">Product 4</Text>
+  </Box>
+  <Box background="blue" border pad="medium" align="center">
+    <Text size="large">Product 5</Text>
+  </Box>
+  <Box background="blue" border pad="medium" align="center">
+    <Text size="large">Product 6</Text>
+  </Box>
+  <Box background="blue" border pad="medium" align="center">
+    <Text size="large">Product 7</Text>
+  </Box>
+</Grid>
+```
+
+We now have a Grid with seven boxes, each containing a product name. The Grid is
+displaying the boxes in a single column, which is the default behavior.
+
+## Add columns
+
+To display the boxes in a grid, we need to add columns. The Grid component has a
+`columns` property that allows you to specify the number and/or width of the
+columns you want to display. Let's add the `columns` property to the Grid and set
+it to `{ count: 3, size: 'auto' }`.
+
+```js
+<Grid columns={{ count: 3, size: 'auto' }}>...</Grid>
+```
+
+Now the boxes are displayed in a grid with three columns.
+
+<StepOne />
+
+## Add product content
+
+Now that we have a grid with three columns, let's add some product content to the
+boxes. Start by adding `ProductCard` and `productList` to App.js. `ProductCard` will be
```

Comment:
Maybe specifying that "ProductCard" is a template/pattern that has been set up for ease of use in this tutorial?